label_sync/labels.yaml

```diff
@@ -701,3 +701,8 @@ repos:
         name: area/prow
         target: both
         addedBy: label
+      - color: 0052cc
```

Comment:
I'm already doing this in #9018

Reply:
Cool, I removed the label_sync change and updated the open issue link to search for the `area/prow/tide` label.